clients/drcachesim/tracer/raw2trace.h

```diff
@@ -439,6 +441,7 @@ class module_mapper_t final {
     byte *last_map_base_ = nullptr;
 
     uint verbosity_ = 0;
+    std::string alt_dir_;
```

Comment:
s/alt_dir_/alt_module_dir_ to be consistent with the rest of the changes.

Reply:
Done.